rfc9000.md

```diff
@@ -4345,7 +4344,7 @@ injection as specified in {{!RFC8201}} and {{Section 5.2 of RFC8085}}.  This
 validation SHOULD use the quoted packet supplied in the payload of an ICMP
 message to associate the message with a corresponding transport connection (see
 {{Section 4.6.1 of DPLPMTUD}}).  ICMP message validation MUST include matching
-IP addresses and UDP ports {{!BCP145}} and, when possible, connection IDs to
+IP addresses and UDP ports {{!RFC8085}} and, when possible, connection IDs to
```

Comment:
Not related to this change, but why is this a normative reference and line 4305 informative?